example/simpleVuePlugin/plugin.js

```diff
@@ -24,7 +24,7 @@ function SimpleVuePlugin() {
             container.appendChild(vm.$mount().$el);
           },
           destroy: function (container) {
-            vm.$destroy();
+            //vm.$destroy();
```

Comment:
Heh, looks like it basically was just [renamed](https://v3-migration.vuejs.org/breaking-changes/)!

> The destroyed lifecycle option has been renamed to unmounted